src/compas/geometry/surfaces/nurbs.py

```diff
@@ -109,25 +144,46 @@ def __data__(self):
 
     @classmethod
     def __from_data__(cls, data):
+        """Construct a Nurbs surface from its data representation.
```

Comment:
is the domain not part of the representation? can it somehow be inferred from another piece of data?